app/navigation/Navigation.scala

```diff
@@ -48,7 +48,9 @@ class Navigation @Inject()(dateTimeHelper: DateTimeHelper,
     WhyWasTheReturnSubmittedLateAgentPage -> ((_, request, _) => routeToMakingALateAppealOrCYAPage(request, CheckMode)),
     ReasonableExcuseSelectionPage -> ((_, request, _) => routeToMakingALateAppealOrCYAPage(request, CheckMode)),
     WhenDidThePersonDiePage -> ((_, request, _) => routeToMakingALateAppealOrCYAPage(request, CheckMode)),
-    OtherRelevantInformationPage -> ((_, request, _) => routeToMakingALateAppealOrCYAPage(request, CheckMode))
+    OtherRelevantInformationPage -> ((_, request, _) => routeToMakingALateAppealOrCYAPage(request, CheckMode)),
+    UploadFirstDocumentPage -> ((_, _, _) => routes.OtherReasonController.onPageLoadUploadList(CheckMode)),
+    UploadAnotherDocumentPage -> ((_,_,_) => routes.OtherReasonController.onPageLoadUploadList(CheckMode))
```

Comment:
For CheckMode, if the user selects No - we go straight back to the CYA page.
if the user selects Yes - we go to add another document passing the mode through